tests/classes/CLI.js

```diff
@@ -291,7 +291,8 @@ describe('CLI', () => {
       process.chdir(this.cwd);
     });
 
-    it('prints general --help to stdout', (done) => {
+    it('prints general --help to stdout', function (done) {
+      this.timeout(10000);
```

Comment:
The `before` and `after` have to be changed from `() =>` to `function (done)` to match this, since it's using `this`.